examples/android/WORKSPACE

```diff
@@ -56,18 +56,26 @@ maven_install(
         "https://maven.google.com",
         "https://repo1.maven.org/maven2",
     ],
+    use_starlark_android_rules = True,
+    aar_import_bzl_label = "@io_bazel_rules_android//rules:rules.bzl",
 )
 
+# Pulling from personal branch since we need a few fixes in rules_android
+# A non complete list can be seen here https://github.com/bazelbuild/rules_android/pulls
 http_archive(
-    name = "build_bazel_rules_android",
+    name = "io_bazel_rules_android",
```

Comment:
Any reason for the name change here? I think that we should keep it consistent so this these repositories aren't downloaded multiple times. And a while back we intentionally switched to `build_bazel_rules_android` to align with the rest of the community and what rules_android recommends.

Reply:
no reason I think I just copied the pattern from rules_kotlin but can switch it back to whatever